src/routes/(console)/project-[region]-[project]/sites/site-[site]/domains/retryDomainModal.svelte

```diff
@@ -58,8 +58,7 @@
             });
             trackEvent(Submit.DomainUpdateVerification);
         } catch (e) {
-            error =
-                'Domain verification failed. Please check your domain settings or try again later';
+            error = e.message ?? 'Domain verification failed. Please check your domain settings or try again later';
```

Comment:
Backend gives custom message here in some flows, like when CAA blocks it. We should always try that message first, as it's very often more specific.